dev/envvars.sh

```diff
@@ -55,7 +55,10 @@ then
 fi
 
 # Configure .NET Core runtime
-export DOTNET_ADDITIONAL_DEPS="${CURDIR}/bin/tracer-home/AdditionalDeps"
+#
+# Not all apps/samples are prepared to run with DOTNET_ADDITIONAL_DEPS - when it is needed define it as
+# export DOTNET_ADDITIONAL_DEPS="${CURDIR}/bin/tracer-home/AdditionalDeps"
```

Comment:
shouldn't we remove setting `DOTNET_ADDITIONAL_DEPS` from `docs/README.md`?

Reply:
Let's do a full review of our scenarios and recommendations around `DOTNET_ADDITIONAL_DEPS`. Following-up via https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/417